pyproject.toml

```diff
@@ -18,7 +18,21 @@ exclude = [
 target-version = "py310"
 
 [tool.ruff.lint]
-extend-select = ["I"]
+select = [
+    # pycodestyle
+    "E",
+    "W",
+    # Pyflakes
+    "F",
```

Comment:
Checking if these (and the corresponding changes to Python) belong in this PR. Per the PR title, I'd only expect `B` adn `SIM` to be added here.

Reply:
Fixed in 4cf75dd14.